<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chooser</class>
 <widget class="QWidget" name="Chooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGradiantMorph">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Gradiant mophologique oublieux</string>
      </attribute>
      <widget class="QPushButton" name="gradiantChooser">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>131</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Choisir la video</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>160</width>
         <height>65</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QSlider" name="alphaChooserGrad">
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelGradAlpha">
          <property name="text">
           <string>alpha</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelGradAlphaValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="image_label">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>40</y>
         <width>291</width>
         <height>251</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>gradiantChooser</zorder>
      <zorder>gridLayoutWidget</zorder>
      <zorder>gridLayoutWidget_5</zorder>
      <zorder>gridLayoutWidget_5</zorder>
      <zorder>image_label</zorder>
     </widget>
     <widget class="QWidget" name="tabMoyArith">
      <attribute name="title">
       <string>Moyenne arithmétique</string>
      </attribute>
      <widget class="QPushButton" name="arithChooser">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>141</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Choisir la video</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabMoyRec">
      <attribute name="title">
       <string>Moyenne récursive</string>
      </attribute>
      <widget class="QPushButton" name="recChooser">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>86</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>160</width>
         <height>65</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QSlider" name="alphaChooserRec">
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelRecAlpha">
          <property name="text">
           <string>alpha</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelRecAlphaValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <zorder>recChooser</zorder>
      <zorder>gridLayoutWidget_2</zorder>
      <zorder>gridLayoutWidget_5</zorder>
     </widget>
     <widget class="QWidget" name="tabMoySD">
      <attribute name="title">
       <string>Moyenne Σ-∆</string>
      </attribute>
      <widget class="QPushButton" name="SDChooser">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>86</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>160</width>
         <height>53</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QSpinBox" name="NChooser">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelSDN">
          <property name="text">
           <string>N</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabSAP">
      <attribute name="title">
       <string>SAP</string>
      </attribute>
      <widget class="QPushButton" name="SAPChooser">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>86</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>160</width>
         <height>122</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="4" column="0">
         <widget class="QSpinBox" name="multipleChooser"/>
        </item>
        <item row="1" column="0">
         <widget class="QSlider" name="alphaChooserSAP">
          <property name="maximum">
           <number>9</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelSAPMultiple">
          <property name="text">
           <string>Multiple</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelSAPAlpha">
          <property name="text">
           <string>alpha</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelSAPAlphaValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
